Adwaita relies on own Metacity theme, 
which is not present in Mate. 
diff -ur gnome-themes-standard-3.22.2-1/themes/Adwaita/index.theme gnome-themes-standard-3.22.2/themes/Adwaita/index.theme
--- gnome-themes-standard-3.22.2-1/themes/Adwaita/index.theme	2018-10-09 11:11:03.646198493 +0000
+++ gnome-themes-standard-3.22.2/themes/Adwaita/index.theme	2018-10-09 11:11:53.559606727 +0000
@@ -150,7 +150,7 @@
 Comment[zh_TW]=只有唯一一個
 Encoding=UTF-8
 GtkTheme=Adwaita
-MetacityTheme=Adwaita
+MetacityTheme=TraditionalOk
 IconTheme=Adwaita
 CursorTheme=Adwaita
 CursorSize=24
diff -ur gnome-themes-standard-3.22.2-1/themes/Adwaita/index.theme.in gnome-themes-standard-3.22.2/themes/Adwaita/index.theme.in
--- gnome-themes-standard-3.22.2-1/themes/Adwaita/index.theme.in	2018-10-09 11:11:03.646331321 +0000
+++ gnome-themes-standard-3.22.2/themes/Adwaita/index.theme.in	2018-10-09 11:11:26.408685032 +0000
@@ -4,7 +4,7 @@
 _Comment=There is only one
 Encoding=UTF-8
 GtkTheme=Adwaita
-MetacityTheme=Adwaita
+MetacityTheme=TraditionalOk
 IconTheme=Adwaita
 CursorTheme=Adwaita
 CursorSize=24
diff -ur gnome-themes-standard-3.22.2-1/themes/Adwaita-dark/index.theme gnome-themes-standard-3.22.2/themes/Adwaita-dark/index.theme
--- gnome-themes-standard-3.22.2-1/themes/Adwaita-dark/index.theme	2018-10-09 11:11:03.675367978 +0000
+++ gnome-themes-standard-3.22.2/themes/Adwaita-dark/index.theme	2018-10-09 11:12:21.608813209 +0000
@@ -150,7 +150,7 @@
 Comment[zh_TW]=只有唯一一個
 Encoding=UTF-8
 GtkTheme=Adwaita
-MetacityTheme=Adwaita
+MetacityTheme=TraditionalOk
 IconTheme=Adwaita
 CursorTheme=Adwaita
 CursorSize=24
diff -ur gnome-themes-standard-3.22.2-1/themes/Adwaita-dark/index.theme.in gnome-themes-standard-3.22.2/themes/Adwaita-dark/index.theme.in
--- gnome-themes-standard-3.22.2-1/themes/Adwaita-dark/index.theme.in	2018-10-09 11:11:03.675500553 +0000
+++ gnome-themes-standard-3.22.2/themes/Adwaita-dark/index.theme.in	2018-10-09 11:12:10.639862220 +0000
@@ -4,7 +4,7 @@
 _Comment=There is only one
 Encoding=UTF-8
 GtkTheme=Adwaita
-MetacityTheme=Adwaita
+MetacityTheme=TraditionalOk
 IconTheme=Adwaita
 CursorTheme=Adwaita
 CursorSize=24
